app/jobs/restart_pipeline_for_sample.rb

```diff
@@ -4,10 +4,10 @@ class RestartPipelineForSample
   def self.perform(sample_id)
     s = Sample.find(sample_id)
     if s.nil?
-      raise Exception("Sample #{sample_id} not found")
+      raise "Sample #{sample_id} not found"
```

Comment:
did you mean to include this file?

Reply:
I realized this code is syntactically wrong and it doesn't work as expected. There is no such thing as `raise Exception` in Ruby. I was probably doing to much Python when I created that code.